yt_dlp/extractor/lastfm.py

```diff
@@ -10,10 +10,10 @@
 class LastFMPlaylistBaseIE(InfoExtractor):
     def _entries(self, url, playlist_id):
         webpage = self._download_webpage(url, playlist_id)
-        start_page_number = int_or_none(self._search_regex(
+        start_page_number = int(self._search_regex(
             r'\bpage=(\d+)', url, 'page', default=None)) or 1
 
-        last_page_number = int_or_none(self._search_regex(
+        last_page_number = int(self._search_regex(
```

Comment:
`last_page_number` is nullable, `int(None)` will raise a non-fatal exception. reverting it back to `int_or_none`